app-office/homebank/Manifest

```diff
@@ -2,3 +2,4 @@ DIST homebank-5.2.2.tar.gz 2601784 BLAKE2B e0a66cebbdc9247aeba9946c4adf5b6677abb
 DIST homebank-5.2.8.tar.gz 2730743 BLAKE2B e5a1cd143f72cdb09b5e097853c89b2dac8ad3e35e0aa5d0f3372350545d887a96ae630cd8a5cb8c605230e84e4402a2c63cef8af76d00ff2c1875aea2313545 SHA512 e1cb0595f1a5cedad83d1b51223d15ea7f37ced9d14edad1f1af8ad8e0eed2938518ed936981558879b5a1bd8aa80d88f4303e69fff0d6e503a8abe61bf872be
 DIST homebank-5.3.2.tar.gz 2906468 BLAKE2B 0672890c469e9580a2128f130230a73d5bdc9e6b50830b1c0ead4f83d7e83837a2df4f50b49218dac9da6eb89f484a6866fcc748e65d5705ba5691e9772b7977 SHA512 777ee682f816f1b3a8f5d018148d79743be9fd17301d16e9c837e9fd46f8bd67cd393bfe9ef2960d87a7066a7f80fa7b88d7826ce62d4542693d3e09d7d5e887
 DIST homebank-5.4.2.tar.gz 2974448 BLAKE2B 0b6f6ad57633ecf560f821295797e9bcc83bb7f69b7b74ad38792f091593b5ae9dc79693b9d44f46414ee3e6f9e9d4fdc2dd5a174f2a50e79177d68506a2235c SHA512 6c28fed7f7a0019be87cd86d37f52a0ddedb0c2752bd311986a4ddfe33913a34e362eadc69e9238e391009fddb45a3a2b59e1176f2ca3f1901293a1b0989223e
+DIST homebank-5.4.3.tar.gz 3014659 BLAKE2B 7ec6aa1c8bdc7d26bae8ae751200a4ad9effd9aac08bd39a1969897d7904468f60606307be1b6e6587a662fd7874c8f73460e559c6d06820056811dd4e8fb85f SHA512 e2b3b45a6d35205c3d2bf24d4c0c065889fa4e7d92cc9551db873b8b33cc15e6c1dd721936b438db0725509233bf265446b0512e23973cbebae7af4a7dce96ba
```

Comment:
Could some of the older versions be removed?

Reply:
I guess so. Do you want me to open a pull request for that? I think it's easier if you just do it.
I'd keep 5.2.8 (current stable), 5.4.2 and 5.4.3

Reply:
Sure I can do it, thanks for letting me know what versions to keep!